include/rppt_tensor_filter_augmentations.h

```diff
@@ -67,6 +67,29 @@ extern "C" {
 RppStatus rppt_box_filter_gpu(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, Rpp32u kernelSize, RpptROIPtr roiTensorPtrSrc, RpptRoiType roiType, rppHandle_t rppHandle);
 #endif // GPU_SUPPORT
 
+/*! \brief Gaussian Filter augmentation on HOST backend for a NCHW/NHWC layout tensor
+ * \details The Gaussian filter augmentation runs for a batch of RGB(3 channel) / greyscale(1 channel) images with an NHWC/NCHW tensor layout.<br>
```

Comment:
We need to add the version changes in this PR in 
- rpp_version.h
- cmakelists.txt
- changelog.md

Please check and add

Reply:
Done